aptos-move/framework/move-stdlib/sources/hash.move

```diff
@@ -3,12 +3,20 @@
 /// The functions in this module are natively declared both in the Move runtime
 /// as in the Move prover's prelude.
 module std::hash {
+    use std::bcs;
+
     // TODO: move sip_hash into aptos_framework
-    native public fun sip_hash<MoveValue>(v: &MoveValue): u64;
-    native public fun sha2_256(data: vector<u8>): vector<u8>;
-    native public fun sha3_256(data: vector<u8>): vector<u8>;
+    public fun sip_hash<MoveValue>(v: &MoveValue): u64 {
```

Comment:
Wait a minute, this thing is in our fork of move-stdlib?!

OMG we should definitely move it into aptos-framework ASAP as the comment suggests

Reply:
Yes... I tried to, but can't get it to work unless someone helps me deal with the 0x01 conflict AND pass `test_genesis_e2e_flow`... See PR text above.

Reply:
Yeah looks like there's an on going discussion about this. Let's try to conclude the current PR and move it in another one.

Reply:
Concluded: moving `sip_hash` from `std::hash` to `aptos_std::aptos_hash` in `aptos-move/framework/aptos-stdlib/sources/hash.move`

Auto-merging once the tests pass!